code/datums/components/heirloom.dm

```diff
@@ -9,16 +12,30 @@
 	owner = new_owner
 	family_name = new_family_name
 
-	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, .proc/examine)
+	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, .proc/on_examine)
 
-/datum/component/heirloom/proc/examine(datum/source, mob/user, list/examine_list)
+/datum/component/heirloom/Destroy(force, silent)
+	owner = null
```

Comment:
isn't strictly needed, but s a good pattern

Reply:
better to have and not need than need than not have, I suppose